### ISSUE TITLE
Fake: [+] [ StackOF  ] Login successful!

### ISSUE THREAD

**Issue by csenger18** [D4Vinci/Cr3dOv3r#63]
When it write out  [+] [ StackOF  ] Login successful! I tested mannually the site is ask for google captcha 
successfully coplete it and go back to StackOF login page. I type credentials again mannually the site says: The email or password is incorrect.

**Comment by Hostn4me** [comment]
I get the same error when checking different Email accounts. It is not uncommon for StackOF to be the only "valid" user/pass combo at least 50% of the time as a matter of fact. Furthermore, I've noticed it returns false positives for other sites as well (saying the user/pass combo matches when they do not).

@D4Vinci  Do you think you will have time to fix this? I can only imagine that you stay busy, and I respect that, but It would definitely be much appreciated and I'll always be willing to beta test.

Thanks in advance.

**Comment by D4Vinci** [comment]
Yup, that was happening with LinkedIn too after 3 wrong attempts so I removed it from the tool.
All I can do for all websites start to ask for google captcha, is to ignore the website once captcha is detected and tell the user.
I think this is the perfect solution for all websites.

**Comment by D4Vinci** [comment]
@Hostn4me The problem with Cr3dov3r is all websites always update their design every while so it will show false positives not only because I'm busy and I am but I will try my best at my spare time again as I'm working on other projects too.

Now, what are the other websites you are talking about?
